app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
+end
```

Comment:
Avoid using rails generators when possible. This will keep your code base smaller, and help you get the practice in!